OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -168,6 +176,115 @@ int main()
 //==============================================================================
 // Test Cases
 //==============================================================================
+void testSmith2016Ligament()
+{
+    using namespace SimTK;
+
+    double mass = 1;
+    double stiffness = 10;
+    double restlength = 0.5;
+    double dissipation = 0.1;
+    double start_h = 0.5;
+
+    // Setup OpenSim model
+    Model osimModel{};
+    osimModel.setName("Smith2016Ligament");
```

Comment:
Can you add a text description of this model so that we don't need to understand the code to understand the model? Many bonus points for an ASCII diagram.